narui_core/src/vulkano_render/vk_util.rs

```diff
@@ -38,13 +30,16 @@ impl VulkanContext {
             ext_debug_report: true,
             ..required_extensions
         };
-        let instance = Instance::new(None, Version::V1_2, &extensions, None)?;
-        std::mem::forget(DebugCallback::new(
-            &instance,
-            MessageSeverity { error: true, warning: true, information: true, verbose: true },
-            MessageType::all(),
-            |msg| log::info!("{}: {}", msg.layer_prefix.unwrap_or("unknown"), msg.description),
-        ));
+        let instance = Instance::new(vulkano::instance::InstanceCreateInfo {
+            enabled_extensions: extensions,
+            ..Default::default()
+        })?;
+        // std::mem::forget(DebugCallback::new(
```

Comment:
Isnt this a handy feature?

I would go for 
```
unsafe {
            std::mem::forget(DebugUtilsMessenger::new(
                instance,
                DebugUtilsMessengerCreateInfo {
                    message_severity: DebugUtilsMessageSeverity::all(),
                    message_type: DebugUtilsMessageType::all(),
                    ..DebugUtilsMessengerCreateInfo::user_callback(Arc::new(|msg| log::info!("{}: {}", msg.layer_prefix.unwrap_or("unknown"), msg.description)))
                }
            ));
        }
```

Reply:
ah, that was the one thing I forgot to port